bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/filter/impl/HttpCacheFilter.java

```diff
@@ -26,7 +26,7 @@
         description = "Intercepts http requests to deal with caching.",
         generateComponent = true,
         generateService = true,
-        order = 0,
+        order = 4999, // Magic number places it immediately after the Granite inner cache
```

Comment:
This pushes the filter up fairly high on the filter stack.. its right after the Granite http inner cache filter